components/page-actions.tsx

```diff
@@ -121,7 +123,7 @@ export default function PageActions({
       .then(() => handleNetworkIssue())
       .catch((err) => console.log("err", err));
   }
-
+  if (!metaMaskWallet) return <></>;
```

Comment:
User needs to understand _why_ it can't see the thing where he pressed -- if he really cant